nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -206,6 +206,9 @@
 - The `services.mxisd` module has been removed as both [mxisd](https://github.com/kamax-matrix/mxisd) and [ma1sd](https://github.com/ma1uta/ma1sd) are not maintained any longer.
   Consequently the package `pkgs.ma1sd` has also been removed.
 
+- `python3.pkgs.etebase-server` has been removed, as it isn't shipped as python package by upstream and no users are known.
```

Comment:
We don't have to add this

Reply:
Thanks for your review! Do you mean we *should* not add this? It seems to a relevant, user-visible change to effectively drop a package?